tests/test_notebooks.py

```
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+"""
+Test all notebooks by executing them with nbconvert.
+"""
+
+import subprocess
+import glob
+import pytest
+
+
+@pytest.mark.parametrize("notebook", glob.glob("**/*.ipynb", recursive=True))
+def test_notebook_executes(notebook):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_notebook_executes`

Add return type annotation: `None`

(ANN201)